concent_api/conductor/service.py

```diff
@@ -113,3 +119,38 @@ def store_frames(
 
 def filter_frames_by_blender_subtask_definition(blender_subtask_definition: BlenderSubtaskDefinition) -> list:
     return list(Frame.objects.filter(blender_subtask_definition=blender_subtask_definition).values_list('number', flat=True))
+
+
+def store_blender_crop_script_parameters(blender_parameters: Any) -> BlenderCropScriptParameters:
+    resolution = ImageResolution(
+        x_dimension=blender_parameters['resolution'][0],
+        y_dimension=blender_parameters['resolution'][1]
+    )
+    resolution.full_clean()
+    resolution.save()
+
+    borders_x = ImageBorders(
+        minimum_border_edge=blender_parameters['borders_x'][0],
+        maximum_border_edge=blender_parameters['borders_x'][1],
+    )
+    borders_x.full_clean()
+    borders_x.save()
+
+    borders_y = ImageBorders(
+        minimum_border_edge=blender_parameters['borders_y'][0],
+        maximum_border_edge=blender_parameters['borders_y'][1],
+    )
+    borders_y.full_clean()
+    borders_y.save()
```

Comment:
You don't need to save these models separately. Django will do it automatically when you save `blender_crop_script_parameters`.

`blender_crop_script_parameters.full_clean()` will probably also run `full_clean()` on the related objects too, though I'm not 100% sure about that.

Reply:
I changed https://github.com/golemfactory/concent/pull/1128#discussion_r269887568.